dangerfile.js

```diff
@@ -17,7 +17,9 @@ const unFlowedFiles = jsFiles.filter(filepath => {
 })
 
 if (unFlowedFiles.length > 0) {
-  warn(`These new JS files do not have Flow enabled: ${unFlowedFiles.join(', ')}`)
+  warn(
+    `These new JS files do not have Flow enabled: ${unFlowedFiles.join(', ')}`,
```

Comment:
For cases like this, you're encouraged to pull the variable bit out into a separate statement.